frontend/src/types/global.d.ts

```diff
@@ -7,6 +7,18 @@ enum UserType {
     None
 }
 
+enum AnswerType {
+    Number,
+    Text,
+    Checkbox,
+    List,
+    User,
+    Leader,
+    Project,
+    Date,
+    Location
```

Comment:
The difference is that for making a Leader-type filter it will send request for leaders list, for Project-type - request for projects list and the same thing for Location-type.